README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
```

Comment:
Lets clear this out so we can add our own info later

Reply:
+1 done =)